json/aerospike/7.1.0.json

```diff
@@ -0,0 +1,3557 @@
+                  "dynamic": false,
+                  "enterpriseOnly": true,
+                  "default": []
+                },
+                "filesize": {
+                  "type": "integer",
+                  "default": 0,
+                  "minimum": 1048576,
+                  "maximum": 2199023255552,
+                  "description": "",
+                  "dynamic": false,
+                  "enterpriseOnly": true
+                },
+                "flush-max-ms": {
+                  "type": "integer",
+                  "default": 1000,
+                  "minimum": 0,
+                  "maximum": 1000,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                },
+                "max-write-cache": {
+                  "type": "integer",
+                  "default": 67108864,
+                  "minimum": 0,
+                  "maximum": 18446744073709551615,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                },
+                "stop-writes-avail-pct": {
+                  "type": "integer",
+                  "default": 5,
+                  "minimum": 0,
+                  "maximum": 100,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                },
+                "stop-writes-used-pct": {
+                  "type": "integer",
+                  "default": 70,
+                  "minimum": 0,
+                  "maximum": 100,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                },
+                "tomb-raider-sleep": {
+                  "type": "integer",
+                  "default": 1000,
+                  "minimum": 0,
+                  "maximum": 4294967295,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                }
+              }
+            }]
+          }
+        }
+      }
+    },
+    "mod-lua": {
+      "type": "object",
+      "additionalProperties": false,
+      "enterpriseOnly": false,
+      "properties": {
+        "cache-enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "",
+          "dynamic": false,
+          "enterpriseOnly": false
+        },
+        "user-path": {
+          "type": "string",
+          "default": "/opt/aerospike/usr/udf/lua",
+          "description": "",
+          "dynamic": false,
+          "enterpriseOnly": false
+        }
+      }
+    },
+    "security": {
+      "type": "object",
+      "additionalProperties": false,
+      "enterpriseOnly": true,
+      "properties": {
+        "default-password-file": {
+          "type": "string",
+          "default": "",
+          "description": "",
+          "dynamic": true,
```

Comment:
Are you sure this is dynamic? I don't see any code to set it from info commands in RC2.

Reply:
ref: https://github.com/citrusleaf/aerospike-server-enterprise/blob/dfa163816c0da78556e5b7c167ff79cfa5987da7/as/src/base/security_info.c#L129

Reply:
Its static.